### ISSUE TITLE
Delete the database from schema_holder table, mysql and postgres in case onboarding fails

### ISSUE THREAD

**Issue by suyashgautam** [Samagra-Development/Text2SQL#47]
Currently, if onboarding fails then in that case a record still persists in the schema_holder table. Also in case the file fails in between then the schema still persists in that particular database type. 
For example - I am onboarding a mysql database and schemaid is "abc" and the file failed while creating a table in that case a record will still be present in schema_holder table and also the mysql db where we are replicating the db.

Make code changes such that we could fix this issue.

**Comment by Deekshithrathod** [comment]
@suyashgautam I can work on this, feel free to assign the issue to me.

**Comment by suyashgautam** [comment]
Sure @Deekshithrathod , I have assigned this issue to you.

**Comment by Deekshithrathod** [comment]
Started working today, I'll reach out if I face any roadblocks

**Comment by Deekshithrathod** [comment]
Hey @suyashgautam, I understand that you might be swamped with work. If you don't mind can you take a look at the message I left a message on discord yesterday? This is the [message](https://discord.com/channels/973851473131761674/1107732361761804329/1128857612285395014)

**Comment by suyashgautam** [comment]
Hi @Deekshithrathod Hope you are doing well. I wanted to check any updates on this ticket?

**Comment by Deekshithrathod** [comment]
@suyashgautam Yes, I finally got help on Discord & set up the project on GitPod yesterday. Narrowed down the scope, i.e. figured out that the changes that need to be made might go [here](https://github.com/Samagra-Development/Text2SQL/blob/main/src/server/db_utils.py#L36). 

Since I'm new to Python, I'm trying to wrap my head around Quart (at least the basics) & make changes. 

Is this blocking any other issues?  I was planning to get to this today or tomorrow.

**Comment by suyashgautam** [comment]
@Deekshithrathod That is good to hear you are able to proceed forward.
It's not a direct blocker but @goyalpramod is currently working on the testing framework where he is working on adding support for sqlite so if the schema is failing continuously Postgres will get cluttered. 
Please create a PR once it is done or you can discuss it on the discord if you are facing any issues.

**Comment by Deekshithrathod** [comment]
@suyashgautam You should un-assign me & give it to someone else. I didn't get time to get to it over the weekend & it seems like I'll be busy this week as well.
